model/src/main/scala/za/co/absa/spline/model/DataLineage.scala

```diff
@@ -42,7 +42,8 @@ case class DataLineage
   operations: Seq[Operation],
   datasets: Seq[MetaDataset],
   attributes: Seq[Attribute],
-  dataTypes: Seq[DataType]
+  dataTypes: Seq[DataType],
+  writeIgnored: Boolean = false
```

Comment:
`writeIgnore` shouldn't be a property of data lineage object. What was wrong with the detecting the fact of ignored result from the metrics?

Reply:
We cannot use metrics for detection for saveAsTable operation. It looks like having this property in DataLineage makes sense.